rust/operator-binary/src/spark_k8s_controller.rs

```diff
@@ -689,7 +670,7 @@ fn spark_job(
         args.push(shutdown_vector_command(VOLUME_MOUNT_PATH_LOG));
     }
 
-    cb.image(spark_image)
+    cb.image(&spark_image.image)
```

Comment:
This can be `image_from_product_image` as well and the extra image_pull_policy(L688) removed.